core/src/validator/__tests__/test.white-space.ts

```diff
@@ -0,0 +1,134 @@
+import { assertEquals } from "@std/assert";
+
+import { Decorator } from "../src/core.ts";
+
+import { KeyWhiteSpace } from "../src/rules/key-white-space.ts";
+
+const mapping = {
+  KeyWhiteSpace: KeyWhiteSpace,
+} as const;
+
+Deno.test("KeyWhiteSpace - withWarning", async () => {
+  const metadata = [
+    {
+      attributes: {
+        foo: "bar ", // should warns for value
+        number_field: 1,
+        nested: {
+          bar: " foo", // should warns for value
+          one: "or",
+          " two": "maybe", // should warns for key
+          three: "who knows    ", // should warns for value
+        },
+      },
+    },
+  ];
+
+  const validatorsReceivedFromFrontend: [keyof typeof mapping] = [
+    "KeyWhiteSpace",
+  ];
+
+  const mainValidator = new Decorator("Main");
+  for (const validator of validatorsReceivedFromFrontend) {
+    mainValidator.Enable(new mapping[validator]());
+  }
+
+  for (const asset_metadata of metadata) {
```

Comment:
normalize variable name casing (here snake_case is used while camelCase is used everywhere else)

Reply:
normalized to camel case everywhere